modular_skyrat/modules/modular_ert/code/pizza/weaponry.dm

```diff
@@ -0,0 +1,16 @@
+/obj/item/knife/hotknife
+	name = "thousand degree knife"
```

Comment:
Shouldn't this be `thousand degrees knife`? I'm actually not sure which is the correct way to spell it.

Reply:
No